README.md

```diff
@@ -78,7 +82,7 @@ cert-specific resources:
 For now, the only implemented provider of the type is
 `katello_ssl_tool`. It works as follow:
 
-1. **generation** - the artefact of this phase is an RPM with the keys
+1. **generation** - the artifact of this phase is an RPM with the keys
```

Comment:
I'm fairly sure `artefact` is correct.

Reply:
https://www.merriam-webster.com/dictionary/artefact
*shrug*